packaging/server/server.dockerfile.dockerignore

```diff
@@ -0,0 +1,40 @@
+# Rust code that isn't need for testbed
+oxidation/client
+oxidation/bindings
+oxidation/generate_test_data
+target
+.cargo
+
+# Editor config folder
+
+.idea
+.vscode
+
+# Python test file & cache folder
+
+tests
+__pycache__
+.hypothesis
+.mypy_cache
+.pytest_cache
+.ruff_cache
+parsec/core
+parsec/*.so
+
+# Cov file
+htmlcov
+
+# Other files
+docs
+json_schema
+logs
+licenses
+newsfragment
+packaging
+!packaging/server/build-backend.sh
```

Comment:
`!packaging/server` ? would avoid silly bug next time we add (or rename) files

Reply:
Done